test/functional/suite_test.go

```diff
@@ -54,23 +54,26 @@ func TestAPIs(t *testing.T) {
 		Groups:   []string{"system:authenticated"},
 	}
 
-	// Allow overriding via environment variable
+	context, err := common.NewTestingContext(cfg)
+	if err != nil {
+		t.Fatalf("\"failed to create testing context: %s", err)
+	}
+	k8sClient = context.Client
+
+	//Allow overriding via environment variable
 	if os.Getenv("BYPASS_STORAGE_TYPE_CHECK") != "true" {
-		// get RMI CR and if cluster storage set to true, skip the suite
-		context, err := common.NewTestingContext(cfg)
-		if err != nil {
-			t.Fatalf("\"failed to create testing context: %s", err)
-		}
-		rhmi, err := common.GetRHMI(context.Client, true)
+		rhmi, err := common.GetRHMI(k8sClient, true)
 		if err != nil {
 			t.Fatalf("error getting RHMI CR: %v", err)
 		}
-		if rhmi.Spec.UseClusterStorage == "true" {
-			t.Skip("Aborting functional tests: \"UseClusterStorage\" is set to true. \nPlease, run another testing suite or reinstall operator with \"UseClusterStorage\" set to false")
+		// For now, we want to allow RHOAM multitenant to be tested on "on-cluster" storage
+		if !rhmiv1alpha1.IsRHOAMMultitenant(rhmiv1alpha1.InstallationType(rhmi.Spec.Type)) {
```

Comment:
If this is for now. At what stage does this get removed.

Reply:
As discussed on a call yesterday, when we are ready with multitenant rhoam and we will want to only test it on byoc clusters. For now, because it is still in development, we want the test to be able to run on non-byoc too.